packages/ember-routing/lib/services/router.js

```diff
@@ -6,6 +6,23 @@ import { shallowEqual, resemblesURL, extractRouteArgs } from '../utils';
    The Router service is the public API that provides component/view layer
    access to the router.
 
+   A simple example of usage:
+
+   ```javascript
+   import Component from '@ember/component';
+   import { inject } from '@ember/service';
+
+   export default Component.extend({
+     router: inject(),
```

Comment:
I actually like to use `import { inject as service } from '@ember/service';`
like in https://guides.emberjs.com/release/applications/dependency-injection/#toc_ad-hoc-injections

Then you can inject like `router: service(),`

Reply:
I updated the PR